frontends/bmd/src/app.tsx

```diff
@@ -76,11 +76,12 @@ export function App({
     }
   }, [internalHardware, screenReader]);
 
-  /* istanbul ignore next - need to figure out how to test this */
-  const onKeyPress = useCallback(
+  const onKeyDown = useCallback(
```

Comment:
Have to use KeyPress to detect keys that do not produce a character, like `F17` from the Accessible Controller.

Reply:
are we sure this change doesn't affect behavior of the accessible controller for other actions?